### ISSUE TITLE
Change color of pencil and cursor when editing

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1193]
**Is your feature request related to a problem? Please describe.**
I prefer things to look as good as possible.  
I saw this in some other app - but no shame in copying **if** it's a good idea!  

**Describe the solution you'd like**
Change color of pencil and cursor when editing to the color of the goal.  
<img width="282" alt="image" src="https://user-images.githubusercontent.com/44887755/236588058-674a26d5-2c4d-41f0-837f-94e0f8d648a9.png">

**Describe alternatives you've considered**
None.

**Additional context**
Might have a negative effect for the yellow color in light theme - but let's try first and evaluate!  
If there's an option to make the cursor a little thicker that would be nice.

**Comment by vinaybadgujar102** [comment]
@tijlleenders I can work on this

**Comment by tijlleenders** [comment]
@vinaybadgujar102 Great!

**Comment by Tushar-4781** [comment]
@vinaybadgujar102  use this website ( https://codepen.io/sosuke/pen/Pjoqqp ) to convert the desired Hex color to a filter and then use that filter in the css of that image

**Comment by vinaybadgujar102** [comment]
#1210 I have updated the code to change the cursor color.
 As I am currently busy with my academic work, the remaining task i.e to change the color of the pencil icon may take some time.

**Comment by tijlleenders** [comment]
@vinaybadgujar102 Thanks a lot!

Your change is already very helpful. Ot has made me realize the cursor is not really visible, I have to zoom in to see the color change. Some investigation is needed what can be done to make this a little thicker.  

Also for me on Amdroid there is an additional teardrop below the cursor. I'm not sure if this can be changed too, but it would make sense.

@Tushar-4781 I propose to merge this contribution and keep the issue open.

@vinaybadgujar102 Good luck with the studies!

**Comment by Tushar-4781** [comment]
> @vinaybadgujar102 Thanks a lot!
> 
> Your change is already very helpful. It has made me realize the cursor is not really visible, I have to zoom in to see the color change. Some investigation is needed what can be done to make this a little thicker.
> 
> Also for me on Amdroid there is an additional teardrop below the cursor. I'm not sure if this can be changed too, but it would make sense.
> 
> @Tushar-4781 I propose to merge this contribution and keep the issue open.
> 
> @vinaybadgujar102 Good luck with the studies!

cool, merge it.

**Comment by tijlleenders** [comment]
Resolved by #1210

**Comment by tijlleenders** [comment]
Somewhere in the last PRs we lost this contribution.

**Comment by Tushar-4781** [comment]
@tijlleenders i think it doesn't exist any more

**Comment by tijlleenders** [comment]
@Tushar-4781 Yeah there is still an edit goal title where this could be added back.
